packages/zoe/src/contracts/scaledPriceAuthority.js

```diff
@@ -22,6 +22,10 @@ import { makeInitialTransform } from '../contractSupport/priceAuthorityInitial.j
  * A contract that scales a source price authority to a target price authority
  * via ratios.
  *
+ * Not durable. Because it only transforms there's nothing important to save.
+ * The facets may sever upon vat restart, but the priceAuthorityRegistry that
+ * holds them can be given a fresh scaling contract to replace them.
```

Comment:
Hm. I wonder how that works. Replacing a price authority in the registry is one thing, but what about the reference from the vaultFactory terms?
https://github.com/Agoric/agoric-sdk/blob/1265a79b4a322a18848c3769e7cc66e2fbffda2d/packages/inter-protocol/src/vaultFactory/vaultManager.js#L171

It makes more sense to me that we could upgrade or restart this contract than wire up a fresh one.